zero-module.yml

```diff
@@ -1,4 +1,4 @@
-name: aws-eks-stack
```

Comment:
👍  didn't realize the other modules depends on `zero-aws-eks-stack`, was wondering why they weren't running haha